README.md

```diff
@@ -14,10 +14,17 @@ and query service management code.
 
 ### Install dependencies
 
-`composer install`
+It is recommened to install the php dependencies via a composer docker container as that contains the required php extensions: 
+```
+docker run --rm -it -v $PWD:/app -u $(id -u):$(id -g) composer install
+```
+
 
-Alternatively you can use docker to run composer
-`docker run --rm --interactive --tty --volume $PWD:/app --user $(id -u):$(id -g) composer install`
+Alternatively, you can run `composer` on your local machine but make sure the `xml` and `curl` php extensions are installed. For example, on Ubuntu:
```

Comment:
If the intention here is to run composer on metal and build the dependencies to be used within a container, the best way is probably to just tell it to ignore any extension or language version requirements.

```suggestion
Alternatively, you can run `composer` on your local machine (ignoring platform requirements):
```

I think ideally we would want a version of the API image that has composer built into it, so that composer uses the same PHP version and environment as the application code. Then we could run it with something like `docker-compose run --rm api composer install`, but that doesn't have to happen now.

Reply:
> I think ideally we would want a version of the API image that has composer built into it, so that composer uses the same PHP version and environment as the application code. Then we could run it with something like `docker-compose run --rm api composer install`, but that doesn't have to happen now.

Cool, that makes sense. In the interim, I noticed that the `composer.json` specifies the PHP version (7.4) which I think helps to ensure the same PHP version is used (environment not so much).
https://github.com/wbstack/api/blob/f29860703fddc3529bf38bc300f7068c3d72efb1/composer.json#L75-L78

Reply:
Yes, but one of the features of `--ignore-platform-reqs` is ignoring this line :|